blablahelp-frontend/src/shared/AuthProvider.tsx

```diff
@@ -0,0 +1,19 @@
+import {createContext, ReactNode, useState} from "react";
+import UserType from "../types/UserType";
+
+interface AuthProviderType {
+    children: ReactNode;
+}
+
+const AuthContext = createContext<any>(null)
+
```

Comment:
Zudem ich die login und logout logik wahrscheinlich auch in den context verschieben, dann dann ist die zusammen und man kann von überall leicht die funktionen nutzen.